public/apps/account/utils.tsx

```diff
@@ -14,22 +14,32 @@
  */
 
 import { HttpStart } from 'opensearch-dashboards/public';
-import { API_AUTH_LOGOUT } from '../../../common';
+import { API_AUTH_LOGOUT, LOCAL_CLUSTER_ID } from '../../../common';
 import { setShouldShowTenantPopup } from '../../utils/storage-utils';
-import { httpGet, httpGetWithIgnores, httpPost } from '../configuration/utils/request-utils';
+import { createRequestContextWithDataSourceId } from '../configuration/utils/request-utils';
 import { API_ENDPOINT_ACCOUNT_INFO } from './constants';
 import { AccountInfo } from './types';
 
 export function fetchAccountInfo(http: HttpStart): Promise<AccountInfo> {
-  return httpGet(http, API_ENDPOINT_ACCOUNT_INFO);
+  return createRequestContextWithDataSourceId(LOCAL_CLUSTER_ID).httpGet({
```

Comment:
Can we replace this context creation with `createEmptyRequestContext()` that represents this idea of not knowing anything about the cluster to be targetted?  This will remove that global and also make a little easier to read the intention of the http request in different scenarios.